Engines/Wine/Verbs/xact/script.js

```diff
@@ -0,0 +1,174 @@
+include(["Engines", "Wine", "Engine", "Object"]);
+include(["Utils", "Functions", "Net", "Resource"]);
+include(["Utils", "Functions", "Filesystem", "Files"]);
+
+/**
+* Inspired from d3dx9 verb, winetricks xact
+* -> https://github.com/Winetricks/winetricks/blob/63bc6dbe612d017a0cb6bf6e4cde265162d75bca/src/winetricks#L9643
+* @returns {Wine}
+*/
+Wine.prototype.xact = function () {
+    var that = this;
+
+    var extractFiles = function(progressBar, filesToExtract, destination, pattern, directory) {
```

Comment:
@plata I think I've seen such methods quite often, could we maybe extract it to remove duplication?

Reply:
Yes, why not. The question is where to put it. Either `Utils/Functions/Filesystem/Extract/` or a new one in `Utils/Functions/Filesystem`.

Reply:
Maybe `archive()` could receive a list?

Reply:
I'm not really sure where to locate such a method, I would follow your advice for this.

Reply:
How about:
```Javascript
new BatchExtract()
    .extractor(new CabExtract())
    .archives(["archive1", "archive2"])
    .extract("arguments")
```

`BatchExtract` would be placed in the same file as `CabExtract`.

Reply:
How do we pass the `progressBar`?

Reply:
```Javascript
new BatchExtract()
    .extractor(new CabExtract())
    .archives(["archive1", "archive2"])
    .wizard(wizard)
    .extract("arguments")
```

Reply:
I merged this PR. Your proposal can be implemented separately. Feel free to provide a PR/open an issue for it.